addons/test_mail/models/test_mail_thread_models.py

```diff
@@ -10,3 +10,21 @@ class MailTestCC(models.Model):
     _inherit = ['mail.thread.cc']
 
     name = fields.Char()
+
+
+class MailTestTrackingValue(models.Model):
+    _name = 'mail.test.tracking.value'
```

Comment:
``mail.test.track.fields.all`` (to match naming of other track related models, mail.test.track.something)